File: app/src/main/java/com/example/potatopaloozac/traveldomainproj/SplashActivity.java
package com.example.potatopaloozac.traveldomainproj;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.potatopaloozac.traveldomainproj.ui.login.LoginActivity;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        Thread t1 = new Thread() {
            @Override
            public void run() {
                super.run();
                try {
                    Thread.sleep(2500);
                } catch (InterruptedException e) {
                    e.printStackTrace();
                }
                Intent i = new Intent(SplashActivity.this, LoginActivity.class);
                startActivity(i);
                finish();
            }
        };

        t1.start();
    }
}

File: app/src/main/java/com/example/potatopaloozac/traveldomainproj/SplashActivity.kt
package com.example.potatopaloozac.traveldomainproj

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        var t1: Thread = Thread() {
            Thread.sleep(2500)

            var i: Intent = Intent(this, MainActivity::class.java)
            startActivity(i)
            finish()
        }

        t1.start()
    }
}
